src/containers/Layout/Layout.tsx

```diff
@@ -188,7 +188,9 @@ const Layout = () => {
           <hr className={styles.divider} />
           {renderUserActions()}
         </Sidebar>
-        <Outlet />
+        <div id="content" className={styles.content} tabIndex={-1}>
+          <Outlet />
+        </div>
```

Comment:
Maybe for later, but perhaps this should be a `<main />` landmark. @langemike?

Reply:
I noticed there are already more occurrences of `<main>` within the app.

I think it's better to apply `<main>` landmark per every page so we have the most flexibility. Eg. search page with filters on the left side, a big (jumbotron) header with a card grid below it etc, user setting page with a panel with links on the side etc.

Yes that means a little bit repetition, but that is the way it is.

We need to create a ticket for this, because not every page currently has a `<main>` landmark.

Reply:
Discussed this with Mike: This can stay a div, but every page does need a `<main>` landmark, we will create a ticket to work on this the upcoming sprint!